tests/test_templates.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime as dt
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
```

Comment:
We will migrate from auth.models.User so the recommended approach is to use `django.contrib.auth.get_user_model() `